app/views/api/v2/models/main.json.rabl

```diff
@@ -1,6 +1,7 @@
 object @model
 
 extends "api/v2/models/base"
+extends "api/v2/layouts/permissions"
```

Comment:
I assumed this would actually be included somewhere generic and not per model. `can_create` is available everywhere. Do we need a guard there?

Reply:
`can_create` has to be in the higher level of the response while `can_edit` and `can_delete` is for each entity.
Unless you know a way to make `can_create` be available only for `HW models`, I am not sure we should make this generic right now.

Reply:
I'm saying the opposite: We should make this available for all models by default. I see, that our models don't inherit from a common layout (bummer!), but maybe we can make this work somehow.

Reply:
My understanding was that this is going to be a PR like any other, e.g. rock-solid. If we just want to experiment, let's please at least prefix all parameters with `experimental_`.
I think this is good quality and it makes a lot of sense to have it, so I'm wondering why we don't go all in.

Reply:
Ok, I'm not against, just that the redmine ticket says models API. If we want to make this available everywhere from day one, I suppose we need to add the common layout for all API endpoints. I'd still prefer to address that one in separate PR that would be merged before this one.

Reply:
Ok, let's agree on doing it for `Models` only for now. We can extend this later.

Reply:
I'm find with not having generic tests as well (for now).

Reply:
> I guess that's a separate task though. My understanding of this PR was PoC that we can handle permissions on one page. If we find no issue, it makes sense to introduce this in the whole API, but once we add it and release new version, there's no way to remove it. Living with this on hw models page is acceptable I think.

Hi @timogoebel @ares 
Was this ever introduced for the whole API?

I would love to use it for subscriptions in Katello.